Modules/Loadable/Markups/MRMLDM/vtkMRMLMarkupsDisplayableManager2D.cxx

```diff
@@ -310,6 +310,11 @@ void vtkMRMLMarkupsDisplayableManager2D
 ::ProcessMRMLNodesEvents(vtkObject *caller,unsigned long event,void *callData)
 {
 
+  if (this->GetMRMLScene()->IsBatchProcessing())
+    {
+    return;
+    }
```

Comment:
Sure, good idea - I can add that.  I was tempted to apply this across the board to all displayable managers but didn't want to make a sweeping change.